packages/repl/src/lib/workers/bundler/index.ts

```diff
@@ -76,6 +76,8 @@ self.addEventListener('message', async (event: MessageEvent<BundleMessageData>)
 						can_use_experimental_async
 					);
 
+					console.log('[bundle worker result]', result);
```

Comment:
is this intentional

Reply:
yes, it's very useful when you're encountering it being stuck in production and then you can just look in the console.